app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+                        <tr>
+                          <th scope="row">Comm:</th>
+                          <td><%= link_to contact.claims_tel_comm, "tel:" + contact.claims_tel_comm %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_tel_tollfree %>
+                        <tr>
+                          <th scope="row">Toll-Free:</th>
+                          <td><%= link_to contact.claims_tel_tollfree, "tel:" + contact.claims_tel_tollfree %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_email %>
+                        <tr>
+                          <th scope="row">Email:</th>
+                          <td><%= link_to contact.claims_email.sub(',', '<br>').html_safe, "mailto:" + contact.claims_email %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_dsn %>
+                        <tr>
+                          <th scope="row">DSN Fax:</th>
+                          <td><%= contact.claims_fax_dsn %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_comm %>
+                        <tr>
+                          <th scope="row">Comm Fax:</th>
+                          <td><%= contact.claims_fax_comm %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_tollfree %>
+                        <tr>
+                          <th scope="row">Toll-Free Fax:</th>
+                          <td><%= contact.claims_fax_tollfree %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_post %>
+                        <tr>
+                          <th scope="row">Address:</th>
+                          <td><%= simple_format(contact.claims_post) %></td>
+                        </tr>
+                      <% end %>
+                    </tbody>
+                  </table>
+                </td>
+              </tr>
+            <%- end -%>
+          </tbody>
+        </table>
+
+        Contact for:
+        <ul>
+        	<li>Lost or Damaged Goods</li>
+          <li>Claims Negotiations with Transporter</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="usa-grid-full customer-service-section" id="retirement-separation">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("retirement-separation.png") %>
+        <h4>Retirement / Separation Extensions</h4>
+      </div>
+      <div class="usa-width-three-fourths">
+        <table>
+          <tbody>
+            <%- @contacts.each do |contact| -%>
+              <% if contact.retiree_dsn || contact.retiree_tel_comm || contact.retiree_tel_tollfree || contact.retiree_email || contact.retiree_fax_dsn || contact.retiree_fax_comm || contact.retiree_fax_tollfree || contact.retiree_post %>
```

Comment:
I felt dirty writing this line, and I'm open to suggestions regarding ways of doing this with less typing. The problem is that the Army does not have any retiree-separatee contact info, but the other four branches have very different contact methods, necessitating checking for the presence of every kind of contact.